IBPSA/Fluid/Movers/FlowControlled_m_flow.mo

```diff
@@ -63,6 +69,13 @@ model FlowControlled_m_flow
     annotation (Placement(transformation(extent={{100,40},{120,60}}),
         iconTransformation(extent={{100,40},{120,60}})));
 
+equation
+  assert(-dp <= dpMax,
+    "In " + getInstanceName() + ": Model operates with head -dp = " + String(-dp) + " Pascals,
+    which exceeds the pressure allowed by the parameter " + getInstanceName() + ".dpMax.
+    This typically happens if the pump or fan forces a high mass flow rate through a closed valve or damper,
+    or if the performance record is entered unreasonable. Please verify your model.");
```

Comment:
```suggestion
    or if the performance record is entered unreasonable. Please verify your model and consider using the model FlowControlled_dp instead of FlowControlled_m_flow.");
```